papers/libceed-paper/paper.rst

```diff
@@ -34,16 +34,16 @@ High-performance operator evaluations with ease of use: libCEED's Python interfa
 Introduction
 ----------------------------------------------------------------------------------
 
-Historically, conventional high-order finite element methods were rarely used for industrial problems because the matrix describing the action of the operator loses sparsity as the order is increased :cite:`Orszag1980`, leading to unaffordable solve times and memory requirements :cite:`brown2010`. This effect typically limited the order of accuracy to at most quadratic, especially because they are computationally advantageous in terms of floating point operations (FLOPS) per degree of freedom (DOF)---see Fig. :ref:`fig-assembledVsmatrix-free`---, despite the fast convergence and favorable stability properties offered by higher order discretizations. Nowadays, high-order numerical methods, such as the spectral element method (SEM)---a special case of nodal p-Finite Element Method (FEM) which can reuse the interpolation nodes for quadrature---are employed, especially with (nearly) affine elements, because linear constant coefficient problems can be very efficiently solved.
+Conventional high-order finite element methods have seen very limited use for industrial problems because the matrix describing the action of the operator loses sparsity as the order is increased :cite:`Orszag1980`, leading to unaffordable solve times and memory requirements :cite:`brown2010`. This effect typically limited the polynomial basis degree to at most quadratic, especially because they are computationally advantageous [TODO: did you mean to say it gave up a lot?] in terms of floating point operations (FLOPs) per degree of freedom (DOF)—see Fig. :ref:`fig-assembledVsmatrix-free`—despite the fast convergence and favorable stability properties offered by higher order discretizations. Nowadays, high-order numerical methods, such as the spectral element method (SEM)—a special case of nodal p-Finite Element Method (FEM) that can reuse the interpolation nodes for quadrature—are employed [TODO: can say they are popular is some domains; cite Nek5000 and MFEM], especially with (nearly) affine elements, because linear constant coefficient problems can be very efficiently solved [TODO: needs to be *separable*; cite sum factorization].
```

Comment:
Thanks @jedbrown I don't understand one of your questions. What did you mean when you said `[TODO: did you mean to say it gave up a lot?] ` Did you refer to the fact that quadratic elements are advantageous in terms of FLOPs per DOFs as explained in the sentence?

Reply:
It limits the degree **because** "they are computationally advantageous"? Something is missing/unclear in this sentence.

Reply:
Thanks. I think I addressed it in [d57be80](https://github.com/valeriabarra/scipy_proceedings/pull/1/commits/d57be804c51c6ff69e303455465d90f31c8fc630)